packages/vaadin-grid/test/scroll-to-index.test.js

```diff
@@ -239,5 +240,33 @@ describe('scroll to index', () => {
       grid.expandedItems = [PARENT];
       grid.scrollToIndex(250);
     });
+
+    it('should not reuse rows if subitems are loaded while scrolling to bottom', (done) => {
+      const grid = fixtureSync(fixtures.treeGrid);
```

Comment:
Please create a `beforeEach` hook under the `describe('Tree grid'` block and move this fixture there.

Reply:
Done.